.gitignore

```
@@ -0,0 +1,8 @@
+# R project files to ignore
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+
+# Mac file to ignore
+.DS_Store
```

You need a new line here!
```suggestion
.DS_Store

```